developers/uploader/uploader-ui.md

```diff
@@ -0,0 +1,49 @@
+# Uploader UI
+
+The Ocean Uploader User Interface, is a powerful web-based tool designed to simplify and streamline the process of managing, uploading, and interacting with digital assets on the Ocean Protocol ecosystem. This intuitive platform offers users a seamless experience for uploading files, obtaining unique identifiers (hashes or CIDs), and manage your decentralized assets.
```

Comment:
Honestly, I think the best place to have steps on how to use the UI is under the user guides and you already did this. 
I would expect that this page presents how any developer can integrate the library into their website and what configuration it can provide instead of steps about how to use it.

Reply:
Got it 👍🏽 pushed some changes to the paragraph. Let me know if now looks better.